pubspec.yaml

```diff
@@ -7,7 +7,7 @@ homepage: https://github.com/dart-lang/dartdoc
 environment:
   sdk: '>=1.14.0 <2.0.0'
 dependencies:
-  analyzer: 0.27.3-alpha.7
+  analyzer: 0.27.4-alpha.8
```

Comment:
It's a bit weird that the command failed even though the proper analyzer version was requested, but based on my quick look at how the dartdoc command is set up this is because we manually point the snapshot to a list of packages not necessarily built from this spec, correct?